docs/src/pages/de/migration/0.21.0.md

```diff
@@ -0,0 +1,227 @@
+
+```markdown
+---
+Layout: '...'
+setup: |
+  import MeineReactKomponente from '../components/MeineReactKomponente.jsx'
+---
+
+# Hydratation bei Sichtbarkeit
+
+<MeineReactKomponente client:visible>
+
+# Hallo Welt!
+
+</MeineReactKomponente>
+```
+
+## Komponenten in Titeln(Frontmatter)
+
+Bisher ließen sich Astro-Komponenten innerhalb des Astro-Frontmatter erzeugen, indem dort JSX-Syntax anstelle der Astro-Syntax verwendet wurde. Dieses Verfahren hatte stets etwas von einem Hack, und dieser kann mit dem neuen Kompilierer nicht weiter unterstützt werden. Wir hoffen, dass wir diese Möglichkeit in einer zukünftigen Version von Astro wieder einführen können - in der wir hierfür jedoch eine andere API, an Stelle der JSX-API, verwenden werden.
+
+Um auf v0.21 umzustellen, konvertiere bitte alle JSX-Astro-Komponenten (dies sind jedwede Astro-Komponenten, die innerhalb des Frontmatter einer anderen Komponente erzeugt werden) in eigenständige Komponenten.
+
+## Umgebungsvariablen
+
+In Astro v21.0 können Umgebungsvariablen aus .env-Dateien in deinem Projektverzeichnis geladen werden.
+
+```ini
+.env                # wird in allen Fällen geladen
+.env.local          # wird in allen Fällen geladen, ignoriert von git
+.env.[mode]         # wird nur im spezifizierten Modus geladen
+.env.[mode].local   # wird nur im spezifizierten Modus geladen, ignoriert von git
+```
+
+Aus Sicherheitsgründen sind ausschließlich Variablen mit dem Prefix `PUBLIC_` für deinen Code verfügbar. 
+
+```ini
+GEHEIMES_KENNWORT=kennwort123
+PUBLIC_IRGENDWER=dieser
+```
+
+In diesem Beispiel ist `PUBLIC_IRGENDWER` verfügbar als ìmport.meta.env.PUBLIC_IRGENDWER für Server- oder Client-seitigen Code, `GEHEIMES_KENNWORT` hingegen nicht.
+
+> In früheren Versionen wurde diesen Variablen das Prefix `SNOWPACK_PUBLIC` vorangestellt, und sie erforderten darüber hinaus das Plugin `@snowpack/plugin-env`.
+
+## Dateiendungen
+
+In Astro v21.0 muss auf Dateien mit Nennung ihrer tatsächlichen Erweiterung verwiesen werden, exakt so wie sie gespeichert sind. 
+
+```tsx
+// Div.tsx
+export default function Div(props) {
+  return <div />;
+}
+```
+
+In diesem Beispiel müsste auf `Div.tsx` mit `Div.tsx` verwiesen werden, nicht mit `Div.jsx`.
+
+```diff
+- import Div from './Div.jsx' // Astro v0.20
++ import Div from './Div.tsx' // Astro v0.21
+```
+
+Diese Änderung betrifft auch Style-Dateien.
+
+```scss
+// Div.scss
+div {
+  all: unset;
+}
+```
+
+```diff
+- <link rel="stylesheet" href={Astro.resolve('./Div.css')}>
++ <link rel="stylesheet" href={Astro.resolve('./Div.scss')}>
+```
+
+## Plugins
+
+In Astro v0.21 können Vite-Plugins innerhalb von `astro.config.mjs` konfiguriert werden.
+
+```js
+import { imagetools } from 'vite-imagetools';
+
+export default {
+  vite: {
+    plugins: [imagetools()],
+  },
+};
+```
+
+Um mehr über Vite-Plugins zu lernen, lies bitte Vites [Plugin Guide](https://vitejs.dev/guide/using-plugins.html).
```

Comment:
_`Vites`_ sounds wrong, maybe saying something along the lines of `lies bitte die [Plugin Guide](https://vitejs.dev/guide/using-plugins.html) von Vite.` would be better?

You'd also need to change Line 191

Reply:
I let the name go altogether, since it is introduced in the subordinate clause. '... lies bitte ihren [Plugin Guide]...' Note that gender must be male here.